projects/ngneat/transloco-messageformat/src/lib/messageformat.module.ts

```diff
@@ -0,0 +1,33 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { TRANSLOCO_TRANSPILER } from '@ngneat/transloco';
+import { TranslocoMessageFormat } from './messageformat.transpiler';
+import { TRANSLOCO_MESSAGE_FORMAT_CONFIG, MessageformatConfig } from './messageformat.config';
+
+export function translocoMessageFormatFactory(config: MessageformatConfig) {
+  return new TranslocoMessageFormat(config);
+}
+
+@NgModule({
+  providers: [
+    {
+      provide: TRANSLOCO_TRANSPILER,
+      useClass: TranslocoMessageFormat
+    }
+  ]
+})
+export class TranslocoMessageFormatModule {
+  static init(config: MessageformatConfig): ModuleWithProviders {
```

Comment:
config => config?

Reply:
If you used the init function it makes sense that you will pass the config, otherwise just don't call the function

Reply:
https://github.com/ngneat/transloco/pull/55#discussion_r320436679